### PR TITLE
Only load Stripe Checkout modal on non-free purchases. #452.

### PR DESCRIPTION
**Pull request by mindctrl**
#452

### CODE REVIEW

File: includes/gateways/class-rcp-payment-gateway-stripe-checkout.php [hunk 1]
```diff
@@ -92,15 +92,19 @@ public function fields() {
 					$level = $form.find('input[name=rcp_level]');
 				}
 
+				var $price = $level.closest('li').find('span.rcp_price').attr('rel') * <?php echo rcp_stripe_get_currency_multiplier(); ?>;
```
Comment (pippinsplugins) [inline includes/gateways/class-rcp-payment-gateway-stripe-checkout.php]: I don't like the idea of relying on the HTML structure here. Can we get rid of `li` and `span` and rely on class names instead?

Comment (mindctrl) [inline includes/gateways/class-rcp-payment-gateway-stripe-checkout.php]: @pippinsplugins agreed. Thought I copied this from another part of RCP somewhere. I'll update it and test.

**Comment by mindctrl** [comment]
@pippinsplugins the push test fails because it's not up to date with master. PR test passed already. Want me to update this branch with master and push it again?

**Comment by pippinsplugins** [comment]
@mindctrl yes please update it

**Comment by mindctrl** [comment]
@pippinsplugins up to date.

**Pull request merged**